recipes/sudospawner/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: sudospawner
+  version: {{ version }}
+
+source:
+  fn: sudospawner-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/sudospawner/sudospawner-{{ version }}.tar.gz
+  sha256: aeb9d228de0ccac1cb56bb7ed79a06e746886836b46797370394c81874c4fa9c
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - jupyterhub >=0.4
+
+test:
+  imports:
+    - sudospawner
+
+about:
+  home: https://github.com/jupyterhub/sudospawner
+  license: BSD-3-Clause
```

Comment:
@parente apparently people will scream now if we do not add the license file. Please see https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L42-L52 for an example.